apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -546,6 +546,10 @@ async function addQuestionAndSave({
     await page.fill('[name="name"]', question.name);
   }
 
+  if (question.disableOnPrefill !== undefined) {
+    await page.click('[name="disableOnPrefill"]');
+  }
```

Comment:
Why this change?
This feature doesn't change the default behaviour so e2e tests shouldn't have a need to be modified.